### PR TITLE
Small doc edits 

### PR DESCRIPTION
**Pull request by DonNateR**
Edited warning at top on installation page.
Edited typo on cloud servers page.

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -4,10 +4,8 @@ Installation and Configuration
 ==============================
 
 .. warning::
-   The installation/configuration instructions here must be
-   considered buyer-beware. As development continues, a range of configuration
-   options will be supported; what works today may not work tomorrow, but it
-   works right now so that's ok.
+   Rackspace CLI is currently in beta. The installation/configurations instructions are subject
+   to change. As development continues, a range of configuration options will be supported.
```
Comment (kenperkins) [inline docs/configuration.rst]: :+1:

File: docs/services/servers.rst [hunk 2]
```diff
@@ -305,12 +305,12 @@ Retrieves details of the specified image::
 
 .. note::
 
-   To guarantee usage of the same image every time, use the `--id` flag. Images often
+   To guarantee usage of the same image every time, use the ``--id`` flag. Images often
```
Comment (kenperkins) [inline docs/services/servers.rst]: why is this double \`\`

**Comment by DonNateR** [comment]
When a new subservice is introduced on the page, I went and changed the markup from ``inline`` to **bold**, to indicate that the section concerns a separate subservice from the one before. I'll go ahead and make this change on each page, as well as look for more typos.

**Comment by kenperkins** [comment]
@DonNateR mentioned this is to make the style look better on http://staging.developer.rackspace.com/docs/rack-cli/services/servers/ where it will ultimately live.

File: docs/services/servers.rst (hunk 2 above)
Comment (kenperkins) [inline docs/services/servers.rst]: because I'm a dummy and didn't realize rst uses double back-ticks for code.

**Comment by DonNateR** [comment]
[rfr]

**Pull request merged**